crates/aggchain-proof-core/src/full_execution_proof.rs

```diff
@@ -28,6 +34,10 @@ pub struct FepPublicValues {
     pub new_withdrawal_storage_root: Digest,
     pub new_block_hash: Digest,
 
+    /// L1 info tree leaf and index containing the `l1Head` as block hash.
+    pub l1_info_tree_leaf: (u32, L1InfoTreeLeaf),
+    /// Inclusion proof of the leaf to the l1 info root.
+    pub l1_head_inclusion_proof: LETMerkleProof<Keccak256Hasher>,
```

Comment:
now that we are adding such data here, I would rename our `FepPublicValues` to `FepInputs` or `FullExecutionProofInputs` instead to be more accurate

and then renaming the `hash()` function to `public_values()` in this new `FepInputs`